addons/web_editor/static/lib/odoo-editor/test/spec/editor.test.js

```diff
@@ -3091,12 +3212,12 @@ X[]
                 await testEditor(OdooEditor, {
                     contentBefore: '<p>[]</p>',
                     stepFunction: async editor => {
-                        insertText(editor, 'a');
-                        insertText(editor, 'b');
-                        insertText(editor, 'c');
+                        await insertText(editor, 'a');
+                        await insertText(editor, 'b');
+                        await insertText(editor, 'c');
```

Comment:
How comes it used to work before ?

Reply:
I'm not sure the awaits are necessary in fact.
I've added them during my debuging to check if it was not the source of one of my problem. 
I just didn't remove them...